packages/apps/transfer/.env.example

```diff
@@ -0,0 +1,10 @@
+NODE_ENV=development
+
+# Kadena Networks
+KADENA_API_TTIL=60000
+KADENA_MAINNET_API=api.chainweb.com
+KADENA_TESTNET_API=api.testnet.chainweb.com
+
+# Gas Setting
+GAS_PRICE="0.00000001"
+GAS_LIMIT="750"
```

Comment:
Not sure what the use cases are that should be covered by default with these gas prices, but isn't this a bit low?
Maybe add a comment what these defaults should represent

Reply:
removed gas price since it's adjustable by user, but we should keep gas limit, and number is coming from old transfer website
let me know if I should change it